### PR TITLE
Curric/landing page header CUR-166

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- Adds Header to Curriculum Landing Page
- Adds SVG to Sanity
- Removes unused previous code
- Add breadcrumb and underline

## Issue(s)

Fixes #CUR-166

## How to test

1. Go to https://deploy-preview-1868--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum

## Screenshots

How it used to look (delete if n/a):

Screenshot 2023-09-05 at 13 50 54

How it should now look:

Screenshot 2023-09-05 at 13 50 16

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by rightsaidjames — APPROVED** [review]
Looks good to me from a QA/Design perspective, pretty much pixel perfect compared to Figma. It's not yet on the Notion board, but as far as I'm concerned LGTM!

**Comment by andykwoods** [comment]
blah

**Comment by andykwoods** [comment]
@k-huggs Thanks for adding Screenshots for @HelenHarlow

**Comment by HelenHarlow** [comment]
This may relate to a different ticket but I notice that with the lot picker component, when it is resized the button is also resized, which cuts off the copy. I think we should aim to have it so the button width stays fixed so this doesn’t happen. Other than that, all looks good to me

**Review by HelenHarlow — APPROVED** [review]

**Comment by k-huggs** [comment]
> This may relate to a different ticket but I notice that with the lot picker component, when it is resized the button is also resized, which cuts off the copy. I think we should aim to have it so the button width stays fixed so this doesn’t happen. Other than that, all looks good to me

Noticed this, it's a regression on the button and what happens when it gets reduced in size. It'll be present across the OWA so will need to be ticketed up too

File: src/components/CMSImage/CMSImage.tsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ const CMSImage: FC<CMSImageProps> = (props) => {
           0,
           0,
           originalDimensions.width,
-          originalDimensions.height,
```
Comment (mantagen) [inline src/components/CMSImage/CMSImage.tsx:129]: Looks like old prettier version here... could you merge in main and locally run `npm run format` and commit the changes?

**Review by mantagen — CHANGES_REQUESTED** [review]
Just the formatting issue -- other than that looks good!

**Review by mantagen — APPROVED** [review]
Awesome!

**Pull request merged**